test/Tests/src/Semantic/Case_Spec.enso

```diff
@@ -88,5 +88,18 @@ spec = Test.group "Pattern Matches" <|
         case Any of
             Any -> Nothing
             _ -> Test.fail "Expected the Any constructor to match."
+    Test.specify "should be able to match on literal values" <|
```

Comment:
Shouldn't the `.enso` test also check for `Cons 1 b` style?

Reply:
Yeap, now it does